### PR TITLE
Update changelog to 1.22.0

### PR DESCRIPTION
**Pull request by mtdowling**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
 # Smithy Changelog
 
+## 1.22.0 (2022-06-14)
+
+### Features
+
+* Added `breakingChanges` property to `@trait` to specify more complex backward compatibility rules. ([#1193](https://github.com/awslabs/smithy/pull/1193))
+* Added automatic casing detection to CamelCaseValidator. ([#1217](https://github.com/awslabs/smithy/pull/1217))
+* Added `--quiet` flag to all CLI commands. ([#]())
```
Comment (kstich) [inline CHANGELOG.md:9]: Needs a PR link.

Comment (gosar) [inline CHANGELOG.md:9]: missing PR

**Review by kstich — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
 # Smithy Changelog
 
+## 1.22.0 (2022-06-14)
+
+### Features
+
+* Added `breakingChanges` property to `@trait` to specify more complex backward compatibility rules. ([#1193](https://github.com/awslabs/smithy/pull/1193))
+* Added automatic casing detection to CamelCaseValidator. ([#1217](https://github.com/awslabs/smithy/pull/1217))
+* Added `--quiet` flag to all CLI commands. ([#](https://github.com/awslabs/smithy/pull/1257))
```
Comment (gosar) [inline CHANGELOG.md:9]:
```suggestion
* Added `--quiet` flag to all CLI commands. ([#1257](https://github.com/awslabs/smithy/pull/1257))
```

**Review by gosar — APPROVED** [review]

**Review by gosar — APPROVED** [review]

**Comment by gosar** [comment]
Closing in favor of https://github.com/awslabs/smithy/pull/1290

**Pull request closed without merging**